Issue #8:

DaniloMarques1 opened the pull request:
Basically adding a time symbol attribute.

With this approach, I am saving the ElapsedTime as seconds or milliseconds, which one is it the TimeSymbol will tell.

Another approach would be to do the math inside the functions that generate the html code. This way we would not need the TimeSymbol attribute and the ElapsedTime attribute would always be in seconds.

DaniloMarques1 committed:
Issue #8:

- adding `TimeSymbol` attribute to `PackageDetails` and  `TestDetails` so that we can use `s` or `ms` depending on the value inside `ElapsedTime`.
- adding `formatTimeDisplay` function that will return either s and the time in seconds or ms and time in milliseconds.

Thatooine commented on main.go:
```diff
@@ -604,3 +617,10 @@ func generatePackageDetailsHTMLElements(testSuiteOverview map[string][]string, p
+
+func formatTimeDisplay(secs float64) (float64, string) {
+	if secs > 1 {
+		return secs, "s"
```
Think we can perhaps truncate to 3 decimal places here. Since the time resolution we care about is only up to millisecond

DaniloMarques1 replied:
Shouldn't we truncate where the seconds/milliseconds are displayed? such as:

`"elapsedTime": fmt.Sprintf("%.3f", testCaseDetails.ElapsedTime)`

Thatooine replied:
That should also do, we just need to insure its being done everywhere where we access **ElapsedTime** compared to when done in a single place

Thatooine commented on main.go:
```diff
@@ -604,3 +617,10 @@ func generatePackageDetailsHTMLElements(testSuiteOverview map[string][]string, p
+	if secs > 1 {
+		return secs, "s"
+	}
+	return secs * float64(1000), "ms"
```
Think we can potentially use the time.ParseDuration function. To convert from seconds to milliseconds. This could potentially remove the long decimal places, of which mostly are just zeros

Thatooine reviewed (changes requested):
Hey DaniloMarques1 I Iike the approach taken to resolve the issue, it was more or less what I had in mind =)

DaniloMarques1 committed:
Issue #8:

- using `time.ParseDuration` to get the correct milliseconds out of the seconds value.
